docs/howto/support/raising-issues.md

```diff
@@ -0,0 +1,111 @@
+# Raising Issues
+
+The Service Center is the Portal used by customers with a paid customer support package tier, usually organisations.
+The portal can be used to create, manage and track issue tickets, and of course to dialogue with our support personnel.
+Customers that are not subscribed to a paid customer support package can issue tickets to us over email.
+
+---
+
+## Understanding issue properties
+
+### Issue Type
+
+One issue property that we often refer to is the "**Issue Type**".
+Generally speaking one could say we only have two issue types, Service Requests and Incidents.
+These are two very different issue types and it is important that we understand in what ways.
+
+#### Service Requests
+
+These could be anything, a request that you wish fulfilled or when you want to report a re-ocurring problem (not an ongoing incident).
+The spectrum of this issue type is very big but some examples are:
+
+- Request for quota increase
+- Request for information
+- Invoice issues
+- Account issues
+- Cloud Management Portal issues
+
+#### Incidents
+
+When you issue a incident ticket with us you are reporting an **ongoing** incident.
+An incident is something that is hindering your normal operation, when an issue with our services is impacting your systems, customers or users.
+
+### Priority Matrix
```

Comment:
I think the information about the priority matrix should move to the bottom of the page. This subsection is not immediately relevant to _filing_ an issue, rather it explains how the issue will be prioritised once it _has_ been filed.

Reply:
This assumption is not entirely correct. We do require the priority to be set by the customer on the issuance of Incident tickets (since the recent update to the portal), so understanding which priority to use is immediately relevant to the filing of an (incident) issue.

It used to be we required the selection of Urgency and Impact and then we set the priority based on the matrix, but we are now flipping that around and having the customer map this.

Reply:
Hmmm! Interesting approach. :) But yes, in that case we'll keep the matrix where it is.